playbooks/openshift-etcd/private/remove-etcdv2-data.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Remove etcd v2 data
-  hosts: oo_etcd_to_config
+  hosts: oo_etcd_to_config[0]
```

Comment:
I'm not sure you can do this.

You could do
```yaml
"{{ groups['oo_etcd_to_config'][0] }}"
```

Reply:
Its a valid form, see https://github.com/openshift/openshift-ansible/blob/master/playbooks/openshift-management/private/uninstall.yml#L3